text_parser.go

```diff
@@ -399,13 +429,18 @@ outer:
 func (p *textPlistParser) parseGNUStepValue() cfValue {
 	typ := p.next()
 	p.ignore()
+
+	if p.peek() == '"' {
+		p.next() // tolerate quoted values
+	}
 	p.scanUntil('>')
 
 	if typ == eof || typ == '>' || p.empty() || p.peek() == eof {
 		p.error("invalid GNUStep extended value")
 	}
 
-	v := p.emit()
+	// tolerate quoted values
+	v := strings.TrimSuffix(p.emit(), "\"")
```

Comment:
For this one, I'd prefer 436 to become

```
p.scanUntilAny(`>"`)
```

and then we can validate with a peek or a next/ignore that we actually got a `">` instead of something malformed like `<*I"1024>`. I think there's some examples of `scanUntilAny` elsewhere in this file

What do you think?

Reply:
Makes sense.

Reply:
Hey! I'm really sorry- I never saw that this comment got edited. That seems like a crazy implementation for a parser, but I'll roll with it. I noticed that they handle a couple other cases differently from us as well, so I'll bring this in with a couple patches.

Sorry for the delay!